src/bundle/Resources/public/js/scripts/fieldType/ezimage.js

```diff
@@ -84,8 +84,9 @@
             const dataContainer = this.fieldContainer.querySelector('.ibexa-field-edit__data');
             const isFileFieldEmpty = fileField.files && !fileField.files.length && dataContainer && !dataContainer.hasAttribute('hidden');
             const { isRequired } = event.target.dataset;
+            const alreadyIsError = this.fieldContainer.classList.contains(this.classInvalid);
```

Comment:
It would be nice to have more price variable name:
```suggestion
            const isInvalidatedByOtherValidator = this.fieldContainer.classList.contains(this.classInvalid);
```